_dev/src/assets/json/translations/en/ui.json

```
@@ -452,7 +452,10 @@
   "smartShoppingCampaignCreation": {
     "allFilters": "All Filters",
     "breadcrumb1": "Campaign",
-    "breadcrumb2": "Create Smart Shopping campaign",
+    "breadcrumb2SSC": "Create Smart Shopping campaign",
+    "breadcrumb3SSC": "Modify Smart Shopping campaign",
+    "breadcrumb2PMax": "Create Performance Max campaign",
+    "breadcrumb3PMax": "Modify Performance Max campaign",
```

I would be ok with the number if they were levels of breadcrumb, but this is not the case here.

What about 

```suggestion
    "breadcrumbCreateSSC": "Create Smart Shopping campaign",
    "breadcrumbEditSSC": "Modify Smart Shopping campaign",
    "breadcrumbCreatePMax": "Create Performance Max campaign",
    "breadcrumbEditPMax": "Modify Performance Max campaign",
```